### DISCUSSION TITLE
403 Forbidden when viewing items

### DISCUSSION THREAD

**Discussion by carbonmedia** [eworkers/K2ForJ4#86] [Q&A] [answered]
Hi, Installed K2ForJ4 on Joomla 3.10.12 site. On the frontend categories view ok, but clicking through to an item produces a 403 error. Any suggestions / help appreciated.
Thanks, Paul

**Comment by eworkers** [comment]
Hello,
Please enable debug in Joomla configuration and set error display to max and share the error again so we can help

**Reply by carbonmedia** [reply, marked as answer]
Hi,
Actually finally figured out what was causing this today. In K2 SEO settings Custom (advanced) SEF settings for K2 URLs was set to Disable. When K2ForJ4 was installed URLs were changed slightly Example "...item/8057-trade..." was changed to "...item/8057:trade..." Once set to enabled the hyphen replaced the colon and the 403 error disappeared.
